docs/scos/dev/system-requirements/202304.0/system-requirements.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'System requirements: Spryker product release 202304.0'
```

Comment:
The document's title must correspond with its file name. Adjust the title or file name accordingly. However, if this file is versioned, then you have only one option - adjusting the title as follows:
```suggestion
title: 'System requirements
```